apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -1134,11 +1141,13 @@ handle_info({force_update_presence, LUser}, StateName,
     NewStateData =
     case StateData#state.pres_last of
         #xmlel{name = <<"presence">>} ->
-            PresenceEl = ejabberd_hooks:run_fold(
+            Stanza = mongoose_stanza:from_kv(packet, StateData#state.pres_last),
```

Comment:
This is exported function, but we don't even have tests for that. It maybe useful sometimes on the other hand I don't remember when I used this recently.